crates/biome_tailwind_parser/src/syntax/variant.rs

```diff
@@ -79,6 +79,10 @@ impl ParseSeparatedList for VariantList {
 }
 
 pub(crate) fn parse_variant(p: &mut TailwindParser) -> ParsedSyntax {
+    if p.at(T![-]) {
+        // variants can't start with a negative sign
+        return Absent;
```

Comment:
Now that `parse_variant` can return `Absent`, shouldn't the caller emit a diagnostic?

Reply:
I don't think so. The lack of a variant existing does not mean there is an error.